client/http_curl.cpp

```diff
@@ -148,6 +149,7 @@ size_t libcurl_read(void *ptr, size_t size, size_t nmemb, HTTP_OP* phop) {
         phop->bytes_xferred += (double)(stRead);
         daily_xfer_history.add(stRead, true);
     }
+    phop->update_http_ip_version();
```

Comment:
This function will write to the log file om every read call. This is not needed.
This should be written only once when connections is established